pkg/controller/pxc/status.go

```diff
@@ -130,6 +132,20 @@ func (r *ReconcilePerconaXtraDBCluster) updateStatus(cr *api.PerconaXtraDBCluste
 			}
 		}
 		cr.Status.Status = cr.Status.PXC.Status
+	case cr.Spec.ProxySQL == nil && cr.Status.PXC.Status == api.AppStateReady:
+		clusterCondition = api.ClusterCondition{
+			Status:             api.ConditionTrue,
+			Type:               api.ClusterReady,
+			LastTransitionTime: metav1.NewTime(time.Now()),
+		}
+		cr.Status.Status = cr.Status.PXC.Status
+	case cr.Spec.ProxySQL != nil && cr.Spec.ProxySQL.Enabled == false && cr.Status.PXC.Status == api.AppStateReady:
+		clusterCondition = api.ClusterCondition{
+			Status:             api.ConditionTrue,
+			Type:               api.ClusterReady,
+			LastTransitionTime: metav1.NewTime(time.Now()),
+		}
+		cr.Status.Status = cr.Status.PXC.Status
```

Comment:
1. r/`cr.Spec.ProxySQL.Enabled == false`/`!cr.Spec.ProxySQL.Enabled`
2. It should be one condition

Reply:
done